redash/query_runner/drill.py

```diff
@@ -0,0 +1,239 @@
+import os
+import logging
+import requests
```

Comment:
Please move the requests import down to the dateutil import to separate it from the stdlib imports.